client/src/components/graph/graph.tsx

```diff
@@ -339,21 +326,7 @@ class Graph extends React.Component<GraphProps, GraphState> {
         ...this.createToolSVG(),
       };
     }
-    /*
-        if the selection tool or state has changed, ensure that the selection
-        tool correctly reflects the underlying selection.
-        */
-    if (
-      currentSelection !== prevProps.currentSelection ||
-      graphInteractionMode !== prevProps.graphInteractionMode ||
-      stateChanges.toolSVG
-    ) {
-      const { tool, container } = this.state;
-      this.selectionToolUpdate(
-        stateChanges.tool ? stateChanges.tool : tool!,
-        stateChanges.container ? stateChanges.container : container!
-      );
-    }
```

Comment:
This may actually be needed in order to delete the lasso on layout change?

Reply:
can confirm that without this - lasso stays on when changing layout, until you click away

Reply:
I'll add this to the lasso ticket